### PR TITLE
zhs

### PR DESCRIPTION
**Pull request by paopaochahuCN**
zhs

### CODE REVIEW

**Comment by paopaochahuCN** [comment]
This is a Chinese translation for Spire Biomes.

File: src/main/resources/anniv6Resources/localization/zhs/ThievesHideout/Monsterstrings.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "偷取活力",
+      "窃取经验",
+      "揭露",
+      "刀刃乱舞"
+    ],
+    "NAME": "贼王",
+    "DIALOG": [
+      "你怎敢闯入我的巢穴！",
+      "一旦其他小偷都没了，你就死定了！",
+      我会偷走你的一切！",
```
Comment (erasels) [inline src/main/resources/anniv6Resources/localization/zhs/ThievesHideout/Monsterstrings.json:21]:
```suggestion
      "我会偷走你的一切！",

```

**Review by erasels — CHANGES_REQUESTED** [review]

**Comment by erasels** [comment]
Thank you, I really appreciate it! Is it alright if I message you regarding additional changes like adding a new zone for example?

**Comment by paopaochahuCN** [comment]
Yes, Thank you!

**Pull request merged**